strawberry/types/info.py

```diff
@@ -79,8 +80,16 @@ def selected_fields(self) -> List[Selection]:
 
     @property
     def context(self) -> ContextType:
+        if isinstance(self._raw_info.context, ContextWrapper):
+            return self._raw_info.context.context
         return self._raw_info.context
 
+    @property
+    def input_extensions(self) -> Dict[str, Any]:
+        if isinstance(self._raw_info.context, ContextWrapper):
+            return self._raw_info.context.extensions
+        return {}
```

Comment:
how come this is called input_extensions? 😊

Reply:
honestly, I just didn't have a better name 😅  I just felt `extensions` is used in so many contexts in strawberry and people might be confused as to which one this is. But I'm happy to change it to whatever you think makes most sense